src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/EventTest.java

```diff
@@ -25,11 +25,12 @@ public void has_correct_values() {
         assertEquals("listCase", Event.LIST_CASE.toString());
         assertEquals("createCaseSummary", Event.CREATE_CASE_SUMMARY.toString());
         assertEquals("revertStateToAwaitingRespondentEvidence", Event.REVERT_STATE_TO_AWAITING_RESPONDENT_EVIDENCE.toString());
+        assertEquals("decisionAndReasonsStarted", Event.DECISION_AND_REASONS_STARTED.toString());
         assertEquals("unknown", Event.UNKNOWN.toString());
```

Comment:
I think the Generate Hearing Bundle one is missing here:
`assertEquals("generateHearingBundle", Event.GENERATE_HEARING_BUNDLE.toString());`